File: app/src/main/java/com/wiktor/weater1/newWeaher/weatherDetalisation/mvp/WeatherContract.java
package com.wiktor.weater1.newWeaher.weatherDetalisation.mvp;

import com.wiktor.weater1.newWeaher.weatherDetalisation.network.response.forecast.WeatherForecastResponse;

import retrofit2.Call;

public interface WeatherContract {
    interface Model {
        Call <WeatherForecastResponse> getForecastWeather(String city, int days);
    }
}

File: app/src/main/java/com/wiktor/weater1/newWeaher/weatherDetalisation/mvp/WeatherContract.kt
package com.wiktor.weater1.newWeaher.weatherDetalisation.mvp

import com.wiktor.weater1.newWeaher.weatherDetalisation.network.response.forecast.WeatherForecastResponse
import io.reactivex.Observable
import retrofit2.Call

interface WeatherContract {
    interface Model {
        fun getForecastWeather(city: String, days: Int): Observable<WeatherForecastResponse>
    }
}